AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -25,7 +25,7 @@ param(
 #   to adopt Azure Files to replace on-premises file servers.
 #   Share level permissions migration cmdlets, used to migrate share level permissions set on
 #   local (on-rem) server  to share on Azure storage.
-
+. ./AzFilesHybridUtilities.ps1

```

Comment:
Suggestion: use [$PSScriptRoot](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.4#psscriptroot). This is an automatic variable (meaning it's set automatically by PowerShell) that tells you what folder the current script is in. 

I'm not 100% sure what path `.` will resolve to if you call the script from another location. Maybe it's fine, but in any case, using `$PSScriptRoot` is the safe way to ensure things get resolved correctly.

Also suggest using `\` over `/`. I think Windows nowadays accepts both, but `\` is the Windows way

```suggestion
. $PSScriptRoot\AzFilesHybridUtilities.ps1
```